gatsby-site/src/components/Navbar/index.js

```diff
@@ -64,6 +58,15 @@ export default class ReactNavbar extends React.Component {
                       </DropdownItem>
                     </DropdownMenu>
                 </UncontrolledDropdown>
+                <NavItem>
+                  <Scrollchor to="partners" className="nav-link">Partners</Scrollchor>
```

Comment:
I made some changes to this in my project-pages refactor PR, but when we're merging we should just keep your version of the navbar scrolling, since there is no project page content right now.